ui/main/src/app/services/processes.service.spec.ts

```diff
@@ -72,20 +72,15 @@ describe('Processes Services', () => {
 
 
     describe('#fetchHbsTemplate', () => {
-            const templates = {
-                en: 'English template {{card.data.name}}',
-                fr: 'Template Français {{card.data.name}}'
-            };
+            const template = 'template {{card.data.name}}';
         it('should return different files for each language', () => {
```

Comment:
Title of test is not valid anymore .

Anyway , remove it as it is not useful

Reply:
Done